crates/rari-cli/main.rs

```diff
@@ -236,6 +239,30 @@ enum Cache {
     None,
 }
 
+/// Delete cache files for all dependencies to force fresh updates
+fn clear_dependency_caches(base_path: &Path) {
```

Comment:
Nit: this isn't truly clearing the cache, but clearing the last check file, so I think it's named a little wrong: perhaps `clear_dependencies_last_checked`, or similar?

Reply:
Fixed in 449ef74.